06-TimeManagement-ModuleRelativeInterval/relative-interval.c

```diff
@@ -0,0 +1,70 @@
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/err.h>
+#include <linux/uaccess.h>
+#include <linux/jiffies.h>
+#include <linux/types.h>
+
+static u64 timestamp;
+
+static ssize_t ro_show(struct class *class,
+		struct class_attribute *attr,
+		char *buf)
+{
+	u64 temp, delta;
+
+	temp = get_jiffies_64();
+	delta = temp - timestamp;
+	timestamp = temp;
+
+	char text[50];
+
+	sprintf(text, "%lld %s %d\n", delta,
```

Comment:
Is the size of variable text enough to accept all possible long long, string and int value?